build-binutils.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (C) 2019 The ClangBuiltLinux Authors
+# Description: Builds a standalone copy of binutils
+
+import argparse
+import multiprocessing
+import os
+import pathlib
+import platform
+import shutil
+import subprocess
+import utils
+
+
+# Convert the host architecture to the first part of a target triple
+def host_to_key():
+    if platform.machine() == "armv7l":
+        return "arm"
+    elif platform.machine() == "ppc64le":
+        return "powerpc64le"
+    elif platform.machine() == "ppc":
+        return "powerpc"
```

Comment:
Consider indexing into a Dict.  Whenever you get these patterns of long if/else chains, consider a switch statement, or indexing into a dictionary/associative array/hashtable/etc.

Reply:
This function also needs a better name.

Reply:
> This function also needs a better name.

Agreed, I just can't think of a good one :(

> Consider indexing into a Dict

Harsh beat me to this one: https://github.com/MSF-Jarvis/tc-build/commit/7a2183fc1231ae2f749ea97b5285b2ab4cbc5670

Reply:
https://github.com/MSF-Jarvis/tc-build/commit/4f5d032a880cca5eb7e52712242e7ff6c6a79675